cookbook/websocket.rst

```diff
@@ -0,0 +1,47 @@
+How to configure Websocket?
+===========================
+
+Sulu comes with a sulu-wide websocket application. This application is optional
+but comes with a few feature which can only be used if the websocket is running.
+
+For example the real-time notification for :doc:`../bundles/collaboration`
+can only be displayed when the websocket is running.
+
+Configuration
+-------------
+
+If you use the config layout of `sulu-standard
+<https://github.com/sulu-io/sulu-standard>`_ you only have to set following
+parameters.
+
+.. list-table::
+
+    * - websocket_url
+      - One url where the websocket is reachable.
```

Comment:
there could be ... if you have multiple domains configured to "find" sulu you could add here one of them. but that might be confusing. i will remove that.